packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -645,6 +653,11 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
         // be achieved through contracts built on top of this contract
         emit WithdrawalFinalized(withdrawalHash, success);
 
+        // Send ETH back to the Lockbox or it'll get stuck here.
+        if (!success) {
+            ethLockbox.lockETH{ value: _tx.value }();
```

Comment:
I don't immediately see an issue here, but inserting a new external call in the withdrawal code path is working thinking through. 

- Are there funky issues related to gas here?
- What should happen if the call fails? Would it be safer to have this as a low level call to avoid blocking execution?

Reply:
We really can't have the call fail here or ETH would get stuck. Basically in the worst case this blocks you from executing failed external calls out of the OptimismPortal which might even be an improvement on the existing behavior... but I don't see any reason why this call would fail.

Reply:
OK, I agree with your assessment here resolving.